src/status_im2/contexts/syncing/standard_authentication/view.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im2.contexts.syncing.standard-authentication.view
+  (:require
+    [quo2.core :as quo]
+    [utils.re-frame :as rf]
+    [react-native.touch-id :as biometric]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.syncing.standard-authentication.enter-password :as enter-password]
+    [react-native.core :as rn]))
+
+(defn authorize
+  [{:keys [set-code use-biometric-auth auth-success auth-fail fallback-button-label]}]
+  (biometric/get-supported-type
+   (fn [biometric-type]
+     (if (and use-biometric-auth biometric-type)
+       (biometric/authenticate
+        {:reason     (i18n/label :t/biometric-auth-confirm-message)
+         :on-success (fn [response]
+                       (auth-success)
+                       (js/console.log "response" response))
+         :on-fail    (fn [error]
+                       (js/console.log "Authentication Failed. Error:" error)
+                       (auth-fail)
+                       (rf/dispatch [:show-bottom-sheet
+                                     {:content (fn [] [enter-password/view set-code])}]))
+         :options    {}})
+       (rf/dispatch [:show-bottom-sheet
+                     {:content (fn [] [enter-password/view set-code fallback-button-label])}])))))
+
+(defn view
+  [{:keys [track-text
+           thumb-color
+           track-text-color
+           customization-color
+           fallback-button-label
+           enter-pass-callback
+           use-biometric-auth
+           on-auth-success
+           on-auth-fail]}]
+  [rn/view {:style {:width "100%"}}
+   [quo/slide-button
+    {:customization-color customization-color
+     :on-complete         #(authorize {:set-code              enter-pass-callback
+                                       :use-biometric-auth    use-biometric-auth
+                                       :auth-success          on-auth-success
+                                       :auth-fail             on-auth-fail
+                                       :fallback-button-label fallback-button-label})
+     :thumb-color         thumb-color
```

Comment:
this seems wrong as we like to encapsulate the colors of the design system components within the component - with the exception of setting cusotomization-color with the `key` from the users choice, e.g `:blue` `:army` etc.

It is better to make those adjustments within the component directly so they do not need to be set.

In this case I checked the design system and there is a discrepancy in these colors between the designs and the syncing flows - so I reached out to design to see which version is correct, as they need to align this 👍 

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=19085-205204&mode=design&t=htgflfZwaUfqzAMW-4

Reply:
I agree @J-Son89

Reply:
ah, so as it turns out this variant using `blur?` being active is not in the component yet. 
I would prefer that we do not hack the component like this and instead it can just be a note to be taken into consideration when the designers are looking at the component.  👍 
I will create the follow up issue to fix the slider button with the `blur` variant.
<img width="291" alt="Screenshot 2023-08-04 at 13 01 12" src="https://github.com/status-im/status-mobile/assets/22799766/1228e170-811f-47d6-aadd-43f1b5b9ce77">

Reply:
Yes, that's why I was passing down the colors. For now I have reverted the changes that I did in the component.